Library/Homebrew/utils/inreplace.rb

```diff
@@ -30,8 +30,6 @@ def initialize(errors)
     #
     # `inreplace` supports regular expressions:
     # <pre>inreplace "somefile.cfg", /look[for]what?/, "replace by #{bin}/tool"</pre>
-    #
-    # @api public
```

Comment:
This should stay.

Reply:
Hmm, my goal was to try to get this to document just the `def inreplace` with yard but for some reason it's not showing up. Poking around at things I _think_ https://github.com/dduugg/yard-sorbet/pull/17 is to blame, so I think this wasn't the issue. I'll address this in a bit.

Reply:
👋  Thanks for the tag. I believe the best practice for hiding individual methods from YARD is to use `# @!visibility private`. Please feel free to file a `yard-sorbet` bug if you're finding things don't work as expected though. 

(I'm not familiar with `brew` internals, but I don't see `yard-sorbet` as an explicit dependency within this repo.)

Reply:
We have an inline rubocop tag `# rubocop:disable Style/OptionalBooleanParameter` on the method and I've confirmed that removing it documents the method correctly. It's not a big deal for now since we should probably fix the areas in the codebase which need to turn the rubocop check off, but `inreplace` is widely used and we'd need to be careful about migrating the last parameter.

Our documentation is generated in a separate repository at https://github.com/Homebrew/rubydoc.brew.sh. Thanks for your work on yard-sorbet, it's really improved out documentation!